alerter.c

```diff
@@ -1,32 +1,50 @@
 #include <stdio.h>
 #include <assert.h>
+#include "alerter.h"
 
+//Variable to holder failure count on alert
 int alertFailureCount = 0;
 
+// Stub function for network alerter
 int networkAlertStub(float celcius) {

```

Comment:
Stub is part of production code in the same file. This will get compiled and get shipped to field along with production code. Hence it is necessary to keep test code and production code separate in order to exclude test code for deployment.

If this is intended to be production code then the method name should be intuitive.